capstone/src/main/java/com/google/sps/servlets/FileDeletionServlet.java

```diff
@@ -80,16 +80,20 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     // This condition below is a second safety belt that ensures the requested
     // file targeted for deletion belongs to the user making the request.
-    if (!deleteUnzippedApk(dataStore, fileName, currentUser.getUserId())) {response.sendError(404);}
+    if (deleteUnzippedApk(dataStore, fileName, currentUser.getUserId())) {
```

Comment:
A lot of these changes feel like they are unrelated to search and should have been in a different PR. Keep your PRs focussed to a specific change - if it's a cleanup that's fine but split that off in the future.

Reply:
Yes Mrs. Anna